### PR TITLE
Remove BETA Warnings

### PR DESCRIPTION
**Pull request by 3d-gussner**
- [X] Make these only English to save some resources
- [X] Fix the DEVEL and ALPHA screen as we added the build number
- [X] New message for BETA which fits nicely on 2 screens instead of 3 with last one only having one word. And rephrase it a bit.
- [X] Disable LCD warning for DEBUG/DEVEL/ALPHA/BETA 

ToDo after merged
- [ ] Cherry-pick to MK3_3.12
  - [ ] Rename `MSG_SPOOL_JOIN` to old name `MSG_AUTO_DEPLETE` in `Firmware/messages.cpp/.h`
  - [ ] Update `lang/po/Firmware.pot` and `lang/po/Firmware_xx.po` files

### CODE REVIEW

File: Firmware/Marlin_main.cpp [hunk 1]
```diff
@@ -832,20 +832,20 @@ void factory_reset()
...
+  #if (FW_DEV_VERSION == FW_VERSION_DEVEL || FW_DEV_VERSION == FW_VERSION_ALPHA)
     lcd_puts_at_P(0, 0, PSTR("Development build !!"));
   #else
     lcd_puts_at_P(0, 0, PSTR("Debbugging build !!!"));
   #endif
     lcd_puts_at_P(0, 1, PSTR("May destroy printer!"));
-    lcd_puts_at_P(0, 2, PSTR("ver ")); lcd_puts_P(PSTR(FW_VERSION_FULL));
-    lcd_puts_at_P(0, 3, PSTR(FW_REPOSITORY));
+    lcd_puts_at_P(0, 2, PSTR("FW")); lcd_puts_P(PSTR(FW_VERSION_FULL));
+    lcd_puts_at_P(0, 3, PSTR("Repo: ")); lcd_puts_P(PSTR(FW_REPOSITORY));
```
Comment (leptun) [inline Firmware/Marlin_main.cpp:848]: Why not use lcd_show_fullscreen_message_and_wait_P also here? You can concatenate strings at compile time and place a single long string to progmem

Comment (3d-gussner) [reply Firmware/Marlin_main.cpp:848]: It is `#if 0` now see latest commit

File: Firmware/Marlin_main.cpp [hunk 2]
```diff
@@ -828,30 +828,31 @@ void factory_reset()
 	}
 	KEEPALIVE_STATE(IN_HANDLER);
 }
-
+#if 0
```
Comment (DRracer) [inline Firmware/Marlin_main.cpp:831]: this saves a lot of code :+1:

**Review by DRracer — APPROVED** [review]
looks correct to me

**Pull request merged**